client/settings/payment-settings/index.js

```diff
@@ -99,21 +102,26 @@ const AccountSettingsDropdownMenu = () => {
 };
 
 const AccountDetailsSection = () => {
-	const accountStatus = wc_stripe_settings_params.accountStatus;
+	const [ isTestModeEnabled ] = useTestMode();
```

Comment:
Same changes as in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2024